src/test_l1_dsVideoPort.c

```diff
@@ -750,13 +751,13 @@ void test_l1_dsVideoPort_positive_dsIsDisplaySurround(void) {
 	}
 
 	// Step 04: Repeat the check for audio surround support
-	for (int i = 0; i < NUM_OF_PORTS; i++) {
+	for (int i = 0; i < gDSvideoPort_NumberOfPorts; i++) {
 		status = dsIsDisplaySurround(handle[i], &isSurroundArray2[i]);
 		UT_ASSERT_EQUAL(status, dsERR_NONE);
 	}
 
 	// Step 05: Compare the array values
```

Comment:
DisplaySurround is need to compare from configuration yaml file 
Please follow the same for other API's

Reply:
Done
for below values there is no correcponding param in yaml..?
1.dsHdcpStatus
2. dsColorDepthCapabilities

Reply:
done